webpack.config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     filename: "[name].[contenthash].js",
     path: path.resolve(__dirname, "integreat_cms/static/dist"),
     clean: true,
-    assetModuleFilename: 'assets/[name]-[hash][ext][query]'
+    assetModuleFilename: 'assets/[name][ext][query]'
```

Comment:
Don't we get caching problems when we remove the hash here? :thinking: 
If we change the files without renaming them, users have to manually clear their cache in order for them to receive the new files...